docs/content/users/quickstart.md

```diff
@@ -318,6 +318,15 @@ In the web browser, log into your account using `admin` and `publish`.
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Joomla
+
+```bash
+mkdir my-joomla-site && cd my-joomla-site
+ddev config --project-type=php
```

Comment:
I think Joomla users are more used to Apache environments so it would be a nice addition switching to apache. Can you please add:

`ddev config --webserver-type apache-fpm`

What do you thinkg?

Reply:
thanks! randy created a commit suggestion based on your suggestion and i just added that to the PR.